projects/ion/src/lib/dropdown/dropdown.component.scss

```diff
@@ -51,6 +51,7 @@
 
```

Comment:
I believe the default fill of the icon is the same as the font. Currently it is being rendered with the neutral-8.